packages/lib/isOutOfBounds.timezone.test.ts

```diff
@@ -0,0 +1,213 @@
+import { describe, expect, it } from "vitest";
+
+import dayjs from "@calcom/dayjs";
+
+import { getRollingWindowEndDate } from "./isOutOfBounds";
+
+const getDayJsTimeWithUtcOffset = ({
+  dateStringWithOffset,
+  utcOffset,
+}: {
+  dateStringWithOffset: string;
+  utcOffset: number;
+}) => {
+  if (!dateStringWithOffset.includes("+")) {
+    throw new Error(
+      "dateStringWithOffset should have a +. That specifies the offset. Format: YYYY-MM-DDTHH:mm:ss+HH:mm"
+    );
+  }
+  return dayjs(dateStringWithOffset).utcOffset(utcOffset);
+};
+
+describe("getRollingWindowEndDate", () => {
```

Comment:
This method has the most important logic in term of ROLLING_WINDOW periodType. So, unit testing it separately.